src/layouts/layout.jsx

```diff
@@ -54,12 +60,26 @@ export const StyledFooterTextTitle = styled.label`
 export const StyledFooterText = styled.label`
   color: #fff;
   size: 17px;
-  line-height: 38px;    
+  line-height: 38px;
 `
 
 function Layout({ children }) {  
   const [contextState, setContextState] = useState(initialState);
   const isMobile = useMediaQuery("(max-width: 768px)");
+
+  const footerSocial = [
+    {path: "https://www.instagram.com/xmartlabs", icon: <InstagramIcon />, id: 'blog'},
+    {path: "https://www.linkedin.com/company/xmartlabs/mycompany/", icon: <Linkedin />, id: 'blog'},
+    {path: "https://es-la.facebook.com/xmartlabs/", icon: <Facebook />, id: 'blog'},
+    {path: "https://twitter.com/xmartlabs", icon: <TwitterIcon />, id: 'blog'},
+  ];
+
+  const footerSocialMobile = [
+    {path: "https://www.linkedin.com/company/xmartlabs/mycompany/", icon: <LinkedInIcon />, id: 'blog'},
+    {path: "https://www.instagram.com/xmartlabs", icon: <InstagramIcon />, id: 'blog'},
+    {path: "https://twitter.com/xmartlabs/", icon: <TwitterIcon />, id: 'blog'},
+    {path: "https://github.com/xmartlabs", icon: <GithubIcon />, id: 'blog'},
```

Comment:
Why do these objects have IDs?

Reply:
Since the base branch corresponds to links, I changed the ID, so I have to rebase the base branch to get the correct values.